docs/installation.md

```diff
@@ -36,10 +36,21 @@ To run it as a standalone process you only need to run the binary file downloade
 
 1. Run installation script:
 
+    Either by piping `curl` straight into `bash`:
+
     ```bash
-    curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh | sudo bash -s -- --installation-token "${SUMOLOGIC_INSTALL_TOKEN}"
+    curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh | sudo -E bash
```

Comment:
Actually, why'd you change the other arguments here?

Reply:
I removed the `--installation-token "${SUMOLOGIC_INSTALL_TOKEN}"` because it's not needed and makes the command unnecessarily longer.

Reply:
The way you've changed it makes it difficult to know how to actually pass arguments to the script when invoked this way. I'd rather it stay the way it is for this reason.

Reply:
Oh right, this is a good argument.

Reply:
Restored the `--installation-token "${SUMOLOGIC_INSTALL_TOKEN}"` part of the installation command.